src/components/App/App.jsx

```diff
@@ -35,30 +34,62 @@ function App() {
                 <Sidebar />
                 <div className="es-content">
                     <Routes>
-                        <Route path="/" element={<Home />} />
+                        <Route
+                            path="/"
+                            element={
+                                <ErrorBoundary>
+                                    <Home />
+                                </ErrorBoundary>
+                            }
+                        />
```

Comment:
Had to give <Home /> it's own ErrorBoundary so that header and footer in other routes wouldn't be erased